File: rangeSumBST_leetcode#938.js
var rangeSumBST = function(root, L, R) {
    let sum = 0;
    let queue = [root];
    while(queue.length){
        let node = queue.shift();
        if(node.val >= L && node.val <= R) sum += node.val;
        if(node.left) queue.push(node.left);
        if(node.right) queue.push(node.right)
    }
    return sum;
};

File: range_sum_bst_leetcode#938.rb
def range_sum_bst(root, l, r)
    sum = 0
    queue = [root]
    while queue.length > 0
        node = queue.shift
        sum += node.val if node.val >= l && node.val <= r
        queue.push(node.left) if node.left
        queue.push(node.right) if node.right
        
    end
    
    sum
end